Example/CoreDiagnostics/Tests/FIRCoreDiagnosticsTest.m

```diff
@@ -328,6 +328,7 @@ - (void)expectEventToBeSentToTransportWithHeartbeat:(BOOL)isHeartbeat {
 
     BOOL isSentEventHeartbeat =
         dataObject.config.sdk_name == logs_proto_mobilesdk_ios_ICoreConfiguration_ServiceType_ICORE;
+    isSentEventHeartbeat &= dataObject.config.has_sdk_name;
```

Comment:
I'm not sure if the style guide mentions this or not, but &&= would be better for a BOOL. this isn't a bitwise operation.

Reply:
I'll post a separate PR for this.